epochStart/bootstrap/process.go

```diff
@@ -201,6 +205,9 @@ func NewEpochStartBootstrap(args ArgsEpochStartBootstrap) (*epochStartBootstrap,
 		maxHardCapForMissingNodes:  args.GeneralConfig.TrieSync.MaxHardCapForMissingNodes,
 		trieSyncerVersion:          args.GeneralConfig.TrieSync.TrieSyncerVersion,
 		enableEpochs:               args.EnableEpochs,
+		dataSyncerFactory:          args.DataSyncerCreator,
+		storerScheduledSCRs:        genericMocks.NewStorerMock("path", 0),
```

Comment:
Right, it shouldn't be a mock here in productive code, but it can be initialized with nil, as it will get updated with the correct storer once the epoch is known.